src/helpers/table_presets.ts

```diff
@@ -0,0 +1,365 @@
+import { _t } from "../translation";
+import { Color } from "../types";
+import { TableConfig, TableStyle } from "../types/table";
+
+export const TABLE_STYLE_CATEGORIES = {
+  none: _t("None"),
+  light: _t("Light"),
+  medium: _t("Medium"),
+  dark: _t("Dark"),
+};
+
+export const DEFAULT_TABLE_CONFIG: TableConfig = {
+  hasFilters: true,
+  totalRow: false,
+  firstColumn: false,
+  lastColumn: false,
+
+  numberOfHeaders: 1,
+
+  bandedRows: true,
+  bandedColumns: false,
+  styleId: "TableStyleMedium2",
+};
+
+interface ColorSet {
+  name: string;
+  coloredText: Color;
+  light: Color;
+  medium: Color;
+  dark: Color;
+  mediumBorder: Color;
+  highlight: Color;
+}
+
+type StyleTemplate = (colorSet: ColorSet) => TableStyle;
+
+const COLOR_SETS = {
```

Comment:
From a functional POV, the colors here and the colors in the color picker should probably be the same. Or at least, the table colors should appear in the "custom colors" as soon as it's used.

We could ask the designer to review those colors to have nice combination.

Reply:
Uh Francois gave me colors from a designer, I forgot to test them 😅

Reply:
A follow UP PR will be to put the same colors in the colorPicker that the ones in the tables. to have some kind of global "theme".